README.md

```diff
@@ -48,19 +48,16 @@ $ git init && git add . && git commit -m "Initial Commit"
 If you do wish to use Docker and Docker Compose, here are some commands you will
 find helpful.
 
-#### Installing Dependencies
-
-Installing dependencies can be performed in a one step command.
-
-```bash
-$ docker-compose run web mix do deps.get, deps.compile
-```
-
```

Comment:
We should make sure the TOC at the top is updated since this is being pulled out.